grid/app/main/dfl/auth/user_session.py

```diff
@@ -0,0 +1,86 @@
+from flask_login import UserMixin
+import syft as sy
+import uuid
```

Comment:
Imports should be grouped in the following order:

1.Standard library imports.
2.Related third party imports.
3.Local application/library specific imports.

You should put a blank line between each group of imports.